### ISSUE TITLE
Quoted arguments are not passed through correctly

### ISSUE THREAD

**Issue by jalada** [pixielabs/cavy-cli#31]
[Related roadmap ticket](https://www.pivotaltracker.com/story/show/172888083)

```
$ cavy run-ios --simulator="iPhone 8"
cavy: Running `react-native run-ios`...
info Found Xcode workspace "CavyTester.xcworkspace"
error Could not find "iPhone" simulator. Run CLI with --verbose flag for more details.
cavy: react-native exited with code 1.

$ cavy run-ios --simulator="iPhone\ 8"
cavy: Running `react-native run-ios`...
info Found Xcode workspace "CavyTester.xcworkspace"
info Launching iPhone 8 (iOS 13.4)
```

**Comment by AbigailMcP** [comment]
Proposed fix https://github.com/pixielabs/cavy-cli/pull/32

**Comment by AbigailMcP** [comment]
Fixed in cavy-cli 2.0
